tools/gen_stubs.zig

```diff
@@ -658,48 +687,80 @@ const blacklisted_symbols = [_][]const u8{
     "__fixunstfdi",
     "__fixunstfsi",
     "__fixunstfti",
+    "__fixunsxfdi",
+    "__fixunsxfsi",
+    "__fixunsxfti",
+    "__fixxfdi",
+    "__fixxfsi",
+    "__fixxfti",
     "__floatdidf",
     "__floatdikf",
     "__floatdisf",
     "__floatditf",
+    "__floatdixf",
     "__floatsidf",
     "__floatsikf",
     "__floatsisf",
     "__floatsitf",
+    "__floatsixf",
     "__floattidf",
     "__floattisf",
     "__floattitf",
+    "__floattixf",
     "__floatundidf",
     "__floatundikf",
     "__floatundisf",
     "__floatunditf",
+    "__floatundixf",
     "__floatunsidf",
     "__floatunsikf",
     "__floatunsisf",
     "__floatunsitf",
+    "__floatunsixf",
     "__floatuntidf",
     "__floatuntisf",
     "__floatuntitf",
+    "__floatuntixf",
```

Comment:
A new compiler-rt export `__floatuntikf` was merged recently that you haven't yet got in your master branch: https://github.com/ziglang/zig/commit/2947a2faab2e287926b20d5d8984ad69bba5bd2c

Worth checking if this will need to be blacklisted, too.

Reply:
Rebased on master and added it to the blacklist, thanks for the catch!